src/control-center/src/main.cpp

```diff
@@ -20,8 +22,11 @@ void signal_handler( int signum ) {
 int main() {
     signal(SIGINT, signal_handler);
 
+    // na razie tutaj, zeby zarejestrowc jeden sensor, potem do osobnego watku chyba
```

Comment:
no tak, ale CC to Twoja część. ja będe w sumie robił dokladnie to samo w sensorze, wiec mozemy jakos razem to poogarniac